### PR TITLE
Add game.push/popScene (aliases to push/popEventContext)

### PR DESCRIPTION
**Pull request by mmoskal**
@eanders-MS feel free to add a little more here - whatever you think is generally useful and you need in kojac. With these two functions my jacdac-multitool (menu system) works fine with game---light

### CODE REVIEW

File: libs/game---light/compat.ts [hunk 1]
```diff
@@ -28,6 +28,18 @@ namespace game {
         return _scene
     }
 
+    export function pushScene() {
```
Comment (eanders-MS) [inline libs/game---light/compat.ts:31]: `pushScene` and `popScene` here might be confusing to some. Are they needed? Kojac calls `push/popEventContext` directly.

Comment (mmoskal) [reply libs/game---light/compat.ts:31]: well, I wanted to have something cross-compatible between game and game---light, but maybe that should not be the goal?

Comment (eanders-MS) [reply libs/game---light/compat.ts:31]: I lean toward keeping it out until there's a clear need. Are there any other missing APIs, or is this the only one?

Comment (mmoskal) [reply libs/game---light/compat.ts:31]: For my use case it was just this one.

I think we should remove game.pushEventContext from game---light and replace it with game.pushScene. This way whatever works in game---light should work the same in standard game, allowing for UI library targeting both.

I don't think there's much risk of confusion here - the Scene class in game---light has almost no fields, and things like Sprites are missing altogether.

Though maybe the UI library could do its own drawing handlers and just use control.*EventContext...

Comment (eanders-MS) [reply libs/game---light/compat.ts:31]: That makes sense.

**Review by eanders-MS — APPROVED** [review]

**Pull request merged**